### DISCUSSION TITLE
Camera photo resolution set, still possible?

### DISCUSSION THREAD

**Discussion by iNakel** [Apparence-io/CamerAwesome#196] [Q&A]
Hello!
I know there have been some changes made to the whole package.
I was wondering... (since I haven't found it), ¿how do I set the resolution of the pictures taken to be a specific size (for example, dimensions: 1920x1080)?

¿Is that feature still possible with all changes?

Thanks, and keep up the great work!

**Comment by fabiotavares** [comment]
I would also like to set a resolution for the capture. It's possible? As?
